frontend/src/components/pages/RegistrantExperience/RegistrationSteps/AdditionalInfo/additionalInfoReducer.tsx

```diff
@@ -53,7 +53,10 @@ export const checkAdditionalQuestionsAnswered = (
   const requiredQuestions = campSpecificFormQuestions
     .filter((question) => question.required)
     .map((question) => question.question);
+
   return campers.every((camper) =>
-    requiredQuestions.every((question) => camper.formResponses?.get(question)),
+    requiredQuestions.every(
+      (question) => camper.formResponses?.get(question) ?? false,
```

Comment:
why do we need the explicit `??` check here

Reply:
Wanted to clarify the behaviour with `undefined`, it should fail